vars/deployCDK.groovy

```diff
@@ -17,6 +19,11 @@ def call(Map conf) {
     if (!conf.bundle_channel) {
         conf.bundle_channel = 'edge'
     }
+    if (!conf.charms_channel) {
+        conf.charms_channel = ""  // is this needed?  will we get a "null" without it?
```

Comment:
Yeah, playing around in a groovy shell suggests that this is needed. Otherwise you'd get `null` rendered in the string below.